parity-util-mem/README.md

```diff
@@ -4,7 +4,6 @@ Collection of memory related utilities.
 
 ## Features
 
-- volatile-erase : Not set by default, `Memzero` struct will be erasing memory through a simple [`write_volatile`](https://doc.rust-lang.org/std/ptr/fn.write_volatile.html) call.
 - estimate-heapsize : Do not use allocator, but `size_of` or `size_of_val`.
```

Comment:
(commenting here because GH doesn't allow me to comment below)

Need to update the `## Dependency` section too now that CoD is removed.